conf/up4.bess

```diff
@@ -244,19 +244,14 @@ if ports[parser.sgi_ifname].nat is not None:
     ue_filter = {"priority": -UEGate,
                  "filter": "ip dst {}".format(ports[parser.sgi_ifname].ext_addrs), "gate": UEGate}
     sgiFastBPF.add(filters=[ue_filter])
-
-# Add Sgi filter rules, i.e.:
-# setting filter to detect gtpu traffic
-# and dst host x.x.x.x                         # check S/PGWU IP
-# and udp dst port 2152                         # check GTPU port
-check_ip = "ip"
-check_spgwu_ip = " and dst host " + \
-    " or ".join(str(x) for x in sgi_ip)
-check_gtpu_port = " and udp dst port 2152"
-GTPUGate = 0 #ports[parser.sgi_ifname].bpf_gate()
-downlink_filter = {"priority": -GTPUGate, "filter": check_ip +
-               check_spgwu_ip + check_gtpu_port, "gate": GTPUGate}
-sgiFastBPF.add(filters=[downlink_filter])
+else:
+    # Add Sgi filter rules, i.e.:
+    # setting filter to detect gtpu traffic
+    # and dst host x.x.x.x                         # check S/PGWU IP
+    # and udp dst port 2152                         # check GTPU port
+    check_ip = "ip"
+    downlink_filter = {"priority": -UEGate, "filter": check_ip, "gate": UEGate}
+    sgiFastBPF.add(filters=[downlink_filter])
```

Comment:
wouldnt this break N9 interface? How would N9 work in a single interface case?

Reply:
Maybe we should rename the blocks s1u/sgi as n3/n6 and add n9 block? in that we can give an cidr option to do filtering to know its n9 helping the single interface case?

Reply:
@krsna1729 , let's do this patch in next PR. Merging the current one as it is now.